appserver/extras/docker-images/basic/pom.xml

```diff
@@ -13,6 +13,6 @@
     <properties>
         <docker.noCache>false</docker.noCache>
         <docker.payara.repository>payara/basic</docker.payara.repository>
-        <docker.keyserver.url>hkp://ipv4.pool.sks-keyservers.net</docker.keyserver.url>
+        <docker.keyserver.url>keyserver.ubuntu.com</docker.keyserver.url>
```

Comment:
Isn't this a bad thing? Looks like we are dependent on one server instead of a pool of servers.
I would keep using the pool instead of a single server

Reply:
It wouldn't build with that keyserver, so I changed it. Unless you have a know of a different pool of servers?

Reply:
How come it built before? There is something missing in your explanation :)

Reply:
I'm not sure, but if you go to sks-keyservers.net it says:
 This service is deprecated. This means it is no longer maintained, and new HKPS certificates will not be issued. Service reliability should not be expected.

Update 2021-06-21: Due to even more GDPR takedown requests, the DNS records for the pool will no longer be provided at all.

It's probably to do with the DNS records

Reply:
My internet search revealed this:

hkp://pgp.mit.edu:11371

https://www.reddit.com/r/archlinux/comments/o5rcs6/psa_you_need_to_update_your_keyserver/

Reply:
Is that any different to just using the ubuntu keyserver? That doesn't look to be a pool either

Reply:
FYI: https://gist.github.com/rjhansen/67ab921ffb4084c865b3618d6955275f
And possible solution: https://github.com/tianon/gosu/issues/39#issuecomment-376028649

Reply:
@Pandrex247 yea you are right. I thought for some reason hkp was a pooled protocol.